packages/model-viewer/src/features/scene-graph.ts

```diff
@@ -300,20 +221,20 @@ export const SceneGraphMixin = <T extends Constructor<ModelViewerElementBase>>(
           return reject('Model missing or not yet loaded');
         }
 
+        // Defaults
         const opts = {
-          // NOTE: automatically include all animations to be exported
-          animations: model.animations,
-          binary: options?.binary || false,
-          trs: options?.trs || true,
-          onlyVisible: options?.onlyVisible || true,
-          maxTextureSize: options?.maxTextureSize || Infinity,
-          forcePowerOfTwoTextures: options?.forcePowerOfTwoTextures || false,
-          includeCustomExtensions: options?.includeCustomExtensions || false,
-          embedImages: options?.embedImages || true,
-          // NOTE: automatically set truncate draw range to true since
-          // we don't expose those parameters
-          truncateDrawRange: true
-        };
+          binary: true,
```

Comment:
@hjeldin I updated this because my clang-format was angry about it. If you use VSCode for development and add the clang-format extension, our repo has a config that it reads to enforce style.

Reply:
Got it, thanks!